src/test/java/com/wikia/webdriver/testcases/mercurytests/HeaderTest.java

```diff
@@ -0,0 +1,123 @@
+package com.wikia.webdriver.testcases.mercurytests;
+
+import com.wikia.webdriver.common.contentpatterns.MercuryMessages;
+import com.wikia.webdriver.common.contentpatterns.MercurySubpages;
+import com.wikia.webdriver.common.contentpatterns.MercuryWikis;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.components.Header;
+import com.wikia.webdriver.elements.mercury.pages.ArticlePage;
+
+import org.testng.annotations.Test;
+
+@Execute(onWikia = MercuryWikis.MERCURY_AUTOMATION_TESTING)
+@InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+public class HeaderTest extends NewTestTemplate {
+
+  @Test(groups = "mercury_header_checkElementsVisibilityWithoutInfobox")
+  public void mercury_header_checkElementsVisibilityWithoutInfobox() {
+    Header header =
+        new ArticlePage()
+            .open(MercurySubpages.NO_INFOBOX)
+            .getHeader();
+
+    Assertion.assertTrue(header.isHeaderVisible(), "header is visible");
+    PageObjectLogging.log("header", MercuryMessages.VISIBLE_MSG, true);
+    Assertion.assertTrue(header.isPageTitleVisible(), "page title is visible");
+    PageObjectLogging.log("page title", MercuryMessages.VISIBLE_MSG, true);
+    Assertion.assertFalse(header.isHeroImageVisible(), "hero image is not visible");
+    PageObjectLogging.log("hero image", MercuryMessages.INVISIBLE_MSG, true);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "hero image" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)